zebra-state/src/service/finalized_state.rs

```diff
@@ -236,6 +260,18 @@ impl FinalizedState {
             );
         }
 
+        // TODO: decide if they will always exist or only when reaching the correct height
+        // (in that case they will turn into Option's)
```

Comment:
I think they should exist from the beginning: 
https://github.com/ZcashFoundation/zebra/issues/2425#issuecomment-884667712